hadoop-tools/hadoop-distcp/src/test/java/org/apache/hadoop/tools/contract/AbstractContractDistCpTest.java

```diff
@@ -758,7 +828,7 @@ public void testDistCpWithUpdateExistFile() throws Exception {
     verifyPathExists(remoteFS, "", source);
     verifyPathExists(localFS, "", dest);
     DistCpTestUtils.assertRunDistCp(DistCpConstants.SUCCESS, source.toString(),
-        dest.toString(), "-delete -update", conf);
+        dest.toString(), "-delete -update" + getDefaultCLIOptions(), conf);
```

Comment:
nit : options can be refactored like it is done above.

Reply:
did think about it, but I also felt that it might be prudent to have tests with the CLI to parse, so make sure that -direct is handled there properly. We've been hit in the past by some failures with s3guard and the FS shell API because we were always working at the API level